ethers-solc/src/compile/project.rs

```diff
@@ -384,6 +397,8 @@ fn compile_parallel(
             .collect::<Result<Vec<_>>>()
     })?;
 
+    // TODO need to do post filtering as the output can contain more files than provided in the
```

Comment:
Wdym post filtering?

Reply:
oh this should have been removed...no longer relevant